plone/namedfile/scaling.py

```diff
@@ -326,4 +326,8 @@ def tag(self,
             direction='thumbnail',
             **kwargs):
         scale = self.scale(fieldname, scale, height, width, direction)
+        if height is not None:
+            kwargs['height'] = height
+        if width is not None:
+            kwargs['width'] = width
```

Comment:
I think here its not the place to handle aspect ratio. This here is low-level, the caller has to care about that. in fact taking care here would i.e. probably break plone.app.imagecropping 
@frisi what do you think?